lib/watir/elements/element.rb

```diff
@@ -493,6 +494,21 @@ def browser
       @query_scope.browser
     end
 
+    #
+    # Returns true if a previously located element is no longer attached to DOM.
+    #
+    # @return [Boolean]
+    # @see Watir::Wait
+    #
+
+    def stale?
+      raise Watir::Exception::Error, "Can not check staleness of unused element" unless @element
```

Comment:
Maybe it should `assert_exists` instead? I can imagine usage of something like this:

```ruby
div = browser.div(id: 'foo')
browser.scroll.to(:end) until div.stale?
```

Do I miss something in the implementation?